scripts/deployment/README.md

```diff
@@ -0,0 +1,72 @@
+# Fornax cluster scripts to add worker nodes and deploy mizar
+
+
+### Update existing masters
+
+- **Step 1.1:  Switch to ROOT user:**
+
+```bash
+sudo su
+```
+
+- **Step 1.2: Go to the deployment script directory and upgrate kernel for mizar**
+
+```bash
+cd /root/go/src/github.com/fornax/scripts/deployment
+./update_kernel.sh
+```
+
+- **Step 1.3: Wait util the master to restart and update the master instance**
+
+```bash
+./update_master.sh
+```
+
+### Create worker nodes
+-	**2 Ubuntu 18.04 VMs **
```

Comment:
** might not be matching here. i believe the intention is to have bold. current way it's not working.

Reply:
Removed the space and make it bold.